meta/NumberOidIndexGenerator.cpp

```diff
@@ -18,6 +18,13 @@ uint64_t NumberOidIndexGenerator::increment()
     return ++m_index;
 }
 
+uint64_t NumberOidIndexGenerator::incrementBy(uint64_t count)
```

Comment:
also fix convention, each param in new line with In prefx

Reply:
Done